### PR TITLE
Ios gestures actions

### PR DESCRIPTION
**Pull request by jlipps**
add special actions for the ios swipe, scroll, and pinch/zoom

### CODE REVIEW

File: ios-gestures/src/main/java/io/testproject/addon/ios_gestures/PinchZoomAction.java [hunk 1]
```diff
@@ -41,6 +41,9 @@
     @Override
     public ExecutionResult execute(IOSAddonHelper helper) throws FailureException {
         IOSDriver<IOSElement> driver = helper.getDriver();
+        if (scale < 1 && velocity > 0) {
```
Comment (vitalybu) [inline ios-gestures/src/main/java/io/testproject/addon/ios_gestures/PinchZoomAction.java:44]: Please add a comment explaining this condition and logic

File: ios-gestures/src/main/java/io/testproject/addon/ios_gestures/SwipeAction.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        summary = "Swipe up, down, left, or right")
+public class SwipeAction implements IOSAction {
+
+    @Parameter(description = "The direction to swipe. Must be 'up', 'down', 'left', or 'right'")
+    public String direction = "";
+
+    @Override
+    public ExecutionResult execute(IOSAddonHelper helper) throws FailureException {
+        IOSDriver<IOSElement> driver = helper.getDriver();
+        driver.executeScript("mobile: swipe", ImmutableMap.of("direction", direction));
```
[the remark this answers is not in the document]
Comment (jlipps) [reply ios-gestures/src/main/java/io/testproject/addon/ios_gestures/SwipeAction.java:41]: yes good idea

File: ios-gestures/src/main/java/io/testproject/addon/ios_gestures/PinchZoomAction.java (hunk 1 above)
Comment (jlipps) [reply ios-gestures/src/main/java/io/testproject/addon/ios_gestures/PinchZoomAction.java:44]: sure thing

**Comment by jlipps** [comment]
ok, comments addressed.

**Pull request merged**